qiskit_nature/second_q/operators/polynomial_tensor.py

```diff
@@ -205,4 +230,49 @@ def transpose(self) -> PolynomialTensor:
         for key, value in self._data.items():
             transpose_dict[key] = np.transpose(value)
 
-        return PolynomialTensor(transpose_dict, self._register_length)
+        return PolynomialTensor(transpose_dict, self._register_length, validate=False)
+
+    def compose(self, other, qargs=None, front=False):
+        # TODO:
+        return self
+
+    def tensor(self, other: PolynomialTensor) -> PolynomialTensor:
+        return self._tensor(self, other)
+
+    def expand(self, other: PolynomialTensor) -> PolynomialTensor:
+        return self._tensor(other, self)
+
+    @classmethod
+    def _tensor(cls, a: PolynomialTensor, b: PolynomialTensor) -> PolynomialTensor:
+        new_data = {}
+        for key in a:
+            if key not in b:
+                continue
+            new_data[key] = np.kron(a[key], b[key])
```

Comment:
I had a `# type: ignore` comment here earlier, but the non-windows CIs complained about an unused type-ignore statement (which I found weird, because I was getting it locally. I figured my mypy version was out-of-sync). Having removed it now, the Windows-CI is failing now: https://github.com/Qiskit/qiskit-nature/actions/runs/3114556050/jobs/5050531078#step:11:13

@manoelmarques any ideas on how to resolve this?